### PR TITLE
Merge pull request #2 from cam5672/main

### PR DESCRIPTION
**Pull request by aditya655**

### CODE REVIEW

File: Public/css/home.css
Comment (cam5672) [inline Public/css/home.css:1]: looks like minor css changes, look good to me

File: Public/css/portfolio.css
Comment (cam5672) [inline Public/css/portfolio.css:1]: same as above added styling for summary table

File: Public/html/home.html [hunk 1]
```diff
@@ -1,53 +1,64 @@
...
-    </main>
+  <!-- Market Overview Section -->
+  <main>
+    <section class="market-container">
+        <h2>Market Overview</h2>
+        <div class="market-charts">
+          <!-- S&P 500 Chart -->
+          <div class="chart">
+            <h3>S&P 500</h3>
+            <canvas id="sp500Chart"></canvas>
```
Comment (cam5672) [inline Public/html/home.html]: we might want to add a button here for how to view the charts?

File: Public/html/home.html [hunk 2]
```diff
@@ -1,53 +1,64 @@
...
+          </div>
+          <!-- NASDAQ Chart -->
+          <div class="chart">
```
Comment (cam5672) [inline Public/html/home.html]: Maybe we condense these? Not sure what your were thinking would be different between the S&P and Nasdaq?

File: Public/html/portfolio.html [hunk 3]
```diff
@@ -1,72 +1,103 @@
...
-    <main>
-        <!-- Add Stock Button -->
-        <button class="add-stock-btn">Add Stock</button>
+  <!-- Portfolio Summary -->
+  <div id="portfolio-summary" class="summary-container">
+    <h2>Portfolio Summary</h2>
+    <p>Total Portfolio Value: $<span id="totalValue">0</span></p>
+    <p>Daily Profit/Loss: $<span id="dailyProfit">0</span></p>
+    <p>Overall Percentage Change: <span id="percentageChange">0</span>%</p>
+  </div>
```
Comment (cam5672) [inline Public/html/portfolio.html:34]: I like this addition

File: Public/html/portfolio.html
Comment (cam5672) [inline Public/html/portfolio.html:1]: Do we want a separate table for recent transactions and portfolio? I think they are duplicative if we can sort but purchase date?

I think the sections should go Summary, Portfolio, watchlist.

**Review by cam5672 — COMMENTED** [review]
Great work @aditya655, really appreciate all you have put into this!

File: Public/html/home.html (hunk 1 above)
Comment (cam5672) [reply Public/html/home.html]: like 5yr, 1 yr, 1 month, 1 day etc.

File: Public/html/portfolio.html [hunk 4]
```diff
@@ -1,72 +1,103 @@
...
+    </table>
+  </div>
 
-    <!-- Footer -->
-    <footer>
-        © 2024 Investment App
-    </footer>
+  <!-- Portfolio Table -->
+  <div class="portfolio-container">
+    <button onclick="showAddStockForm()">Add Stock</button>
```
Comment (cam5672) [inline Public/html/portfolio.html]: Maybe centered or below the my portfolio section?

File: Public/html/portfolio.html (hunk 3 above)
Comment (aditya655) [reply Public/html/portfolio.html:34]: thanks

File: Public/html/portfolio.html (hunk 4 above)
Comment (aditya655) [reply Public/html/portfolio.html]: Ok will do.

File: Public/html/home.html (hunk 2 above)
Comment (aditya655) [reply Public/html/home.html]: Do you want me to have a radio button option to display both graphs or just have one? Overlay? Just wondering.

**Pull request merged**